.jenkins/Libraries/vars/helpers.groovy

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+def CmakeArgs(String build_type = "RelWithDebInfo", String code_coverage = "OFF", String debug_malloc = "ON", String lvi_args="", String cmake_args = "") {
+    def args = "-G Ninja -DCMAKE_INSTALL_PREFIX:PATH='/opt/openenclave' -DCPACK_GENERATOR=DEB -DCODE_COVERAGE=${code_coverage} -DUSE_DEBUG_MALLOC=${debug_malloc} -DCMAKE_BUILD_TYPE=${build_type} ${lvi_args} ${cmake_args} -Wdev"
```

Comment:
Improvement: can directly return the string instead of first defining a new variable